src/blog/2023/08/UNS-article.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Why the Automation Pyramid blocks digital transformation - The Role of Unified Namespace
+subtitle: A Critical Examination of the Automation Pyramid's Obstruction to Digital Transformation
+description: This article analyzes the Automation Pyramid's constraints and explains the Unified Namespace, highlighting its potential to evolve digital transformation in manufacturing.
+date: 2023-08-08 
+authors: ["marian-demme"]
+#image: blog/2023/07/images/opc-ua-3/opc-ua-3-title-image.png
+tags:
+    - posts
+---
+A few years ago, I wrote an article, in German, detailing my understanding of how the Automation Pyramid, a widely adopted reference model for the IT landscape of manufacturing firms, is essentially hindering digital transformation. Now, as conversations around the Unified Namespace (UNS) and particular frameworks continue to evolve, I revisit my earlier notions, review the latest updates to reference frameworks and update my article.
+<!--more-->
+
+## The Pyramid’s Dilemma
+
+The Automation Pyramid is grounded in the standard ISA-95, which aligns with IEC 62264 and DIN EN 62264. It delineates the functional hierarchy within a manufacturing enterprise. Over 25 variations of the Automation Pyramid exist in academic literature, all of them fundamentally mapping to the same core concept, tracing back to the Computer-integrated manufacturing (CIM)-Pyramid of the 1970s. Although ISA-95 does not explicitly refer to a pyramid, it introduces five functional hierarchical levels often visualized as a pyramid.
```

Comment:
You should link to these specifications.

Reply:
[Added Links](https://github.com/flowforge/website/pull/1009/commits/78064c33395969115b6d9e8b6950ed503bcca21a)